packages/domain/lib/model/get_data_response.dart

```diff
@@ -0,0 +1,9 @@
+class GetDataResponse {
+  final Map<String, dynamic> headers;
+  final List<dynamic> body;
```

Comment:
in our case, this model is only used for lists. so list can be list<Movie>, not dynamic